packages/textlint-rule-preset-lmc/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@lmc-eu/textlint-rule-preset-lmc",
+  "version": "0.0.0",
```

Comment:
I believe the first release ever should be `0.1.0` but that's probably covered, isn't it?

Reply:
Thanks, fixed.